app/dashboard/templates/dashboard/components/participant-card.html

```diff
@@ -16,96 +16,51 @@
 
 {% endcomment %}
 {% load i18n static %}
-<div v-for="user in users" class="card card-user shadow-sm border-0 bg-lightblue" :key="user.id">
-  <div class="d-flex flex-column py-1">
-    <button class="btn position-absolute align-self-end" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false"><i class="fas fa-ellipsis-v"></i></button>
-    <div class="dropdown-menu dropdown-menu-right font-caption bg-white">
-      <a :href="`/funding/new?invite=${ user.id }`" target="_blank" class="dropdown-item p-2">
-        <i class="fab fa-ethereum mr-2"></i>
-        Fund Bounty
-      </a>
-      <div class="dropdown-divider"></div>
-      <a :href="`/profile/${ user.handle }`" class="dropdown-item p-2" target="_blank">
-        <i class="fas fa-user mr-2"></i>
-        View profile
-      </a>
-    </div>
-    <a :href="`/profile/${ user.handle }`" class="mx-auto">
-      <img class="rounded-circle shadow-sm mt-n5 mb-3 bg-white" width="113" height="113" :src="`/dynamic/avatar/${ user.handle }`" alt="${user.handle} Avatar"/>
-    </a>
-    <h5 class="text-center card-user_name">
-      [[ user.name || user.handle ]]
-      <span v-if="user.verification">
-        <img src="{% static 'v2/images/badge-verify.svg' %}" width="15" height="15" title="Verified user" alt="verify badget">
-      </span>
-    </h5>
-    <a :href="`/profile/${ user.handle }`" v-bind:data-usercard="[[ user.handle ]]" class="text-center d-block">
-      @[[ user.handle ]]
+<div v-for="user in users" class="card border-bottom-0 border-left-0 border-right-0 border-top pt-4" :key="user.id">
```

Comment:
Rather than using `card` and dropping the borders, could we use `border-top-1 border-grey` instead? I feel like the rounded corners make it look odd?

Reply:
Oh i'm not sure why it was built like that but updating it 

![image](https://user-images.githubusercontent.com/5358146/113111220-979c9a00-9225-11eb-8f80-0cb28749c461.png)